app/src/main/res/layout/activity_nfc_scanning.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".sample.MainActivity">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/teal_400"
+        app:layout_constraintTop_toTopOf="parent"
+        app:title="@string/nfc_scanning_title"
+        app:titleTextColor="@color/white" />
+
+    <ProgressBar
+        android:id="@+id/prgLoading"
```

Comment:
Since we're not using ViewBinding, how about following this scheme?
```suggestion
        android:id="@+id/pNfcScanningLoading"
```
or
```suggestion
        android:id="@+id/p_nfc_scanning_loading"
```

Reply:
Or we can add `ViewBinding` to the project if it makes life simpler. Your call.

Reply:
Isn't it `pbLoading`?

Reply:
But as we don't need to make it long like the other ids, so I think just `pbLoading` is okay.